weblate/accounts/forms.py

```diff
@@ -421,7 +421,7 @@ class ContactForm(forms.Form):
         label=gettext_lazy("Message"),
         required=True,
         help_text=gettext_lazy(
-            "Please contact us in English, otherwise we might "
+            "Please contact us in English. Otherwise we might "
             "be unable to process your request."
```

Comment:
```suggestion
            "find ourselves unable to process your request."
```

Reply:
This needs to be as simple as possible, so even people with basic English skills get it. No need to complicate it or make it wordy.